bin/mv/mv.c

```diff
@@ -319,6 +319,12 @@ err:		if (unlink(to))
 	 */
 	preserve_fd_acls(from_fd, to_fd, from, to);
 	(void)close(from_fd);
+
+	ts[0] = sbp->st_atim;
+	ts[1] = sbp->st_mtim;
+	if (futimens(to_fd, ts))
+		warn("%s: set times", to);
```

Comment:
Do we potentially want to do this before copying ACLs as well?

Reply:
I haven't played with extended attributes or ACL's yet, so I don't know...